app/dashboards/archive_dashboard.rb

```diff
@@ -0,0 +1,46 @@
+require "administrate/base_dashboard"
+
+class ArchiveDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    created_at: Field::DateTime,
+    updated_at: Field::DateTime,
+    status: Field::String,
+    file: Field::HasOne
+  }.freeze
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = [
+    :id,
+    :created_at,
+    :updated_at,
+    :status
+  ].freeze
+
+  # SHOW_PAGE_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's show page.
+  SHOW_PAGE_ATTRIBUTES = [
+    :id,
+    :created_at,
+    :updated_at,
+    :status,
+    :file
+  ].freeze
+
+  # Overwrite this method to customize how users are displayed
+  # across all pages of the admin dashboard.
+  #
+  def display_resource(archive)
+    "Archive : #{archive.try(:file).try(:byte_size)}"
```

Comment:
Il y a plus dans l'appli de

```suggestion
    "Archive : #{archive&.file.&byte_size}"
```

Mais y'a peut etre une différence subtile que je ne connais pas